pulseapi/entries/views.py

```diff
@@ -47,6 +47,13 @@ def post_validate(request):
 
     return True
 
+class EntriesPagination(PageNumberPagination):
+    """
+    Add support for pagination and custom page size
+    """
+    page_size = 50
```

Comment:
Not really. Suggestion by @mmmavis. Actually I might make it 48 since that's divisible by 2 and 3 nicely, for even load more layouts